phys2bids/tests/test_bids_units.py

```diff
@@ -0,0 +1,12 @@
+from phys2bids.bids_units import bidsify_units
+
+
+def test_bidsify_units():
```

Comment:
I have a doubt:
The first two asserts should be failing.
In fact, they should be:
```
    assert bidsify_units('V') == 'V'
    # test unit with standard prefix
    assert bidsify_units('centik') == 'cK'
```

Reply:
I know what you mean, let me check why they are not, actually i could just modify them

Reply:
wait, no they shouldn't

Reply:
they are actually in lowercase as that the way they come out of the function, remember?

Reply:
we could make the function output the unit in upper case

Reply:
I think that changing the way `new_unit` is updated (see suggestion) should solve the problem.
The unit is not always in upper case (for instance, `mol`, `s`, or `Hz`)